src/cli/near.rs

```diff
@@ -300,11 +599,201 @@ pub async fn execute_command<T: AsRef<str>>(
             WriteCommand::FactoryUpdate { wasm_bytes_path } => {
                 let args = std::fs::read(wasm_bytes_path).unwrap();
                 let tx_outcome = client
+                    // I cannot find this engine method called as "factory_update"
```

Comment:
It's here: https://github.com/aurora-is-near/aurora-engine/blob/develop/engine/src/lib.rs#L339

It only exists on testnet currently.

Reply:
ok